kedro-telemetry/README.md

```diff
@@ -20,42 +20,39 @@ for further information on the intent of the data collection and what data is co
 For technical information on how the telemetry collection works, you can browse
 [the source code of `kedro-telemetry`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-telemetry).
 
-## How do I consent to the use of Kedro-Telemetry?
+## How do I withdraw consent?
```

Comment:
I would suggest changing the order of paragraphs and moving **How is the data collected** above the current one, so the withdrawal paragraph is at the bottom.

Reply:
Done